app/javascript/controllers/redirect_controller.js

```diff
@@ -0,0 +1,11 @@
+import {Controller} from "@hotwired/stimulus"
+
+export default class extends Controller {
+  static targets = ["redirecting"]
+  static values = {url: String}
+
+  connect() {
+    this.redirectingTarget.remove()
+    window.location = this.urlValue
```

Comment:
Triple checking, but we can't use `Turbo.visit(this.urlValue)` here, right?

Reply:
I tried it with Turbo again, I was able to make it work with that. I am not sure what I was running into, but it worked this time.

Reply:
Tried it in multiple browsers, all working.